### PR TITLE
Update registerSchemesAsPrivileged for Electron>5.x

### PR DESCRIPTION
**Pull request by lacymorrow**
Related #238 .

This allows me to run and package Electron 6 (without signing).

### CODE REVIEW

**Comment by lacymorrow** [comment]
This should be bumped, the appveyor tests timed out

**Comment by lacymorrow** [comment]
once more

File: skeleton/app.js [hunk 1]
```diff
@@ -17,6 +17,14 @@ import Squirrel from './squirrel'; // DEPRECATED
 const { app, BrowserWindow, dialog } = electron;
 const { join } = path;
 
+if (parseInt(process.versions.electron.split('.')[0]) >= 5) {
```
Comment (darqs) [inline skeleton/app.js]: Can you check if your code is still needed? I think code from App.contructor (line 31-37) should resolve your problem.

**Comment by lacymorrow** [comment]
Thanks @darqs!
Moving forward I would be interested in becoming a collaborator if you and @wojtkowiak are open to the idea 😄

**Comment by LDubya** [comment]
hi @lacymorrow, were you ever able to get it to run with signing? I can get electron v6 to run, but the app crashes on startup after signing in Apple's sandboxed mode for me.

**Comment by lacymorrow** [comment]
I have been doing my best to move away from this project as I don't see support from the author and have been unable to get collaborators rights to make necessary updates.

I'm helping to support https://github.com/saltyshiomix/nextron and also can wholeheartedly recommend https://github.com/sindresorhus/electron-boilerplate. Both are up-to-date and have build processes around Electron-Builder (the build-tool in this project)

**Comment by SxMShaDoW** [comment]
It's a little sad :(. I feel your pain as well. We built an app that utilises meteor and electron and this has become a pain. It seems like they are trying their best, but aren't as responsive as we need. I'm hoping they open it up to people that want to help. Migration off of meteor would be a huge pain for us. We have had to code around bugs (like setTimeout) which are fixed in later versions of the app.

**Pull request closed without merging**